src/lib.rs

```diff
@@ -8,6 +8,13 @@
 // At your choosing (See accompanying files LICENSE_APACHE_2_0.txt,
 // LICENSE_MIT.txt and LICENSE_BOOST_1_0.txt).
 
+use std::{
+    pin::Pin,
+    task::{Context, Poll},
+};
```

Comment:
I think importing the prelude from pasts should let you remove these imports, just need to replace instances of `Context` to `Exec`